src/configured_subscribed_notifications.c

```diff
@@ -0,0 +1,520 @@
+ /**
+ * @file configured_subscribed_notifications.c
+ * @author Jeremie Leska <jeremie.leska@6wind.com>
+ * @brief ietf-subscribed-notifications configured subscribed notifications
+ *
+ * @copyright
+ * Copyright 2023 6WIND S.A.
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#define _GNU_SOURCE
+
```

Comment:
Missing includes. You have used `ATOMIC_INC_RELAXED`, which is in `compat.h`. But even for standard functions, `sprintf` is in `stdio.h` so please add all those headers.